back-end/insertDB.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*
+
+# input : list organized by python dictionary
+# output : print (insert success or fail)
+
+import test_list
+import pymysql
+
+attributes = ['name', 'address', 'phone_number', 'latitude', 'longitude', 'rating', 'monday_open_hours', 'tuesday_open_hours', 'wednesday_open_hours', 'thursday_open_hours', 'friday_open_hours', 'saturday_open_hours', 'sunday_open_hours', 'photourl', 'take_out_available', 'parking_available', 'bike_parking_available', 'good_for_groups', 'ambience', 'wi_fi_available']
```

Comment:
I see 'name' used at least twice. It is not recommended to use repeated hardcoded strings. Can you create a variables for the repeated hardcoded strings?

Reply:
I realized that the list 'attributes' is not used. So, I deleted it.

Reply:
It doesn't have any repeated hardcoded strings after deleting the list 'attributes'. So It is used only once. Is it better to create variables then? Or just leave as it is?

Reply:
How do you think? Would you create a variable or keep this as is? (I just want to understand your thoughts on this subject. I don't want you to just apply my suggestion without thinking about it logically)

Reply:
I think i don't have to make it because It only used once :) 
If I use the same words more than 2, I'll create variables.

Reply:
Sounds good to me :)